qbplatform/quote/models.py

```diff
@@ -13,19 +15,59 @@ class Tournament(models.Model):
 
     name = models.CharField(max_length=500)
 
+    def __str__(self):
+        return self.name
+
 
 class TournamentLocation(models.Model):
 
     tournament = models.ForeignKey(Tournament, on_delete=models.CASCADE)
     date = models.DateField()
     location = models.CharField(max_length=500, null=True)
 
+    def __str__(self):
+        return self.location + " site of tournament: " + str(self.tournament)
+
 
 class Packet(models.Model):
 
     name = models.CharField(max_length=500)
     tournament = models.ForeignKey(Tournament, on_delete=models.CASCADE)
 
+    def __str__(self):
+        return "Packet " + self.name + " in tournament: " + str(self.tournament)
+
+#   Model of a per-packet distribution
+#       has a one-to-many relationship with DistributionCategory
+class Distribution(models.Model):
+
+    num_tossups = models.PositiveSmallIntegerField(null=True, default=20)
+    num_bonuses = models.PositiveSmallIntegerField(null=True, default=20)
+
+    # a distribution is unique to a tournament/question set 
```

Comment:
This seems reasonable but it would also be useful to have some pre-baked distributions that people can load (i.e. I can select "ACF Regionals" or "ACF Nationals" or "CO Trash" and get the corresponding top-level distribution).

Reply:
Yeah, I agree. But those will be entered and available sitewide after the app is built.